### PR TITLE
Make signed in Teacher navigation for Non-english users same as English Users

### PR DESCRIPTION
**Pull request by juanmanzojr**
Makes signed in Teacher navigation for Non-english users the same as English Users

### Signed in Teacher English
Screenshot 2023-10-04 at 9 43 47 AM

### Signed in Teacher Non-english
Screenshot 2023-10-04 at 9 43 06 AM

## Links

- jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?modal=detail&selectedIssue=ACQ-983)

## Testing story
Local and UI Tests

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/ui/features/foundations/header.feature [hunk 1]
```diff
@@ -2,22 +2,24 @@
 @single_session
 Feature: Header navigation bar
 
-Scenario: Signed out user in English should see 6 header links
```
Comment (TurnerRiley) [inline dashboard/test/ui/features/foundations/header.feature]: Would you mind merging in staging please so it's clearer which changes are specific to this PR?

Comment (TurnerRiley) [reply dashboard/test/ui/features/foundations/header.feature]: Oh sorry I meant merging in the signed-out changes (I figured those were already in staging, my bad)

Comment (juanmanzojr) [reply dashboard/test/ui/features/foundations/header.feature]: Ok should be good now, sorry I merged incorrectly the first time

**Review by TurnerRiley — APPROVED** [review]
Great job!

**Pull request merged**